packages/react-next/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -17,135 +18,124 @@ const LARGE_IMAGE_SIZE = 71;
-      imageSize,
-      focused,
-    });
-
-    const { className, ...nativeProps } = getNativeProps<{ className: string }>(rest, inputProperties);
-
+export const ChoiceGroupOptionBase = (props: IChoiceGroupOptionProps) => {
+  const {
+    ariaLabel,
+    focused,
+    required,
+    theme,
+    iconProps,
+    imageSrc,
+    imageSize = { width: 32, height: 32 },
+    disabled,
+    // eslint-disable-next-line deprecation/deprecation
+    checked,
+    id,
+    styles,
+    name,
+    ...rest
+  } = props;
+
+  const classNames: IProcessedStyleSet<IChoiceGroupOptionStyles> = getClassNames(styles!, {
+    theme: theme!,
+    hasIcon: !!iconProps,
+    hasImage: !!imageSrc,
+    checked,
+    disabled,
+    imageIsLarge: !!imageSrc && (imageSize!.width > LARGE_IMAGE_SIZE || imageSize!.height > LARGE_IMAGE_SIZE),
+    imageSize,
+    focused,
+  });
+
+  const { className, ...nativeProps } = getNativeProps<{ className: string }>(rest, inputProperties);
+
+  const onRenderLabel = (): JSX.Element => {
     return (
-      <div className={this._classNames.root}>
-        <div className={this._classNames.choiceFieldWrapper}>
-          <input
-            aria-label={ariaLabel}
-            id={id}
-            className={css(this._classNames.input, className)}
-            type="radio"
-            name={name}
-            disabled={disabled}
-            checked={checked}
-            required={required}
-            {...nativeProps}
-            onChange={this._onChange}
-            onFocus={this._onFocus}
-            onBlur={this._onBlur}
-          />
-          {onRenderField(this.props, this._onRenderField)}
-        </div>
-      </div>
+      <span id={props.labelId} className="ms-ChoiceFieldLabel">
+        {props.text}
+      </span>
     );
-  }
-
-  private _onChange = (evt: React.FormEvent<HTMLInputElement>): void => {
-    const { onChange } = this.props;
-    if (onChange) {
-      onChange(evt, this.props);
-    }
-  };
-
-  private _onBlur = (evt: React.FocusEvent<HTMLElement>) => {
-    const { onBlur } = this.props;
-    if (onBlur) {
-      onBlur(evt, this.props);
-    }
-  };
-
-  private _onFocus = (evt: React.FocusEvent<HTMLElement>) => {
-    const { onFocus } = this.props;
-    if (onFocus) {
-      onFocus(evt, this.props);
-    }
   };
 
-  private _onRenderField = (props: IChoiceGroupOptionProps): JSX.Element => {
-    const { id, imageSrc, imageAlt = '', selectedImageSrc, iconProps } = props;
+  const renderField = (): JSX.Element => {
+    const { imageAlt = '', selectedImageSrc } = props;
 
-    const imageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
+    const currentImageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
 
-    const onRenderLabel = props.onRenderLabel
-      ? composeRenderFunction(props.onRenderLabel, this._onRenderLabel)
-      : this._onRenderLabel;
+    const renderedLabel = props.onRenderLabel
```

Comment:
`renderedLabel` isn't really a good name for this since it's a function, not the label itself.

Also I wonder if we could change `composeRenderFunction` to accept undefined for the custom renderer (and if it's undefined, just return the default renderer) and get rid of this extra conditional everywhere.